src/mlpack/core/optimizers/sga/sga_update.hpp

```diff
@@ -18,7 +18,15 @@ namespace mlpack {
 namespace optimization {
 
 /**
- * Update policy for Stochastic Gradient Ascent (SGA).
+ * Update policy for Gradient Ascent.
+ * @article{Kingma2014,
+ *   author  = {R.S. Sutton, David McAllester, Satinder Singh, Yishay Mansour},
+ *   title   = {Policy Gradient Methods for Reinforcement Learning with 
+                Function Approximation },
+ *   journal = {NIPS},
+ *   year    = {2000},
+ *   url     = {https://goo.gl/UwfMnd}
```

Comment:
I think we can add the official url:
https://papers.nips.cc/paper/1713-policy-gradient-methods-for-reinforcement-learning-with-function-approximation.pdf
Check whether this is official or not please. I have just tried to find that out.